astropy/cosmology/tests/test_parameter.py

```diff
@@ -139,15 +152,21 @@ def test_Parameter_derived(self, cosmo_cls, all_parameter):
             all_parameter.name not in cosmo_cls.__parameters__
         )
 
+    def test_Parameter_default(self, cosmo_cls, all_parameter):
+        """Test :attr:`astropy.cosmology.Parameter.default`."""
+        assert hasattr(all_parameter, "default")
+        assert isinstance(
+            all_parameter.default, (Sentinel, type(None), float, u.Quantity)
+        )
+
     # -------------------------------------------
     # descriptor methods
 
     def test_Parameter_descriptor_get(self, cosmo_cls, cosmo, all_parameter):
         """Test :attr:`astropy.cosmology.Parameter.__get__`."""
         # from class
         parameter = getattr(cosmo_cls, all_parameter.name)
-        assert isinstance(parameter, Parameter)
-        assert parameter is all_parameter
+        assert np.all(parameter.default == all_parameter.default)
```

Comment:
Test shouldn't be using `assert np.all(...)`. They should be using either [`np.testing.assert_array_equal()`](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html), which produces better output if the test fails, or `assert all(...)` if `assert_array_equal()` can't be used.

Reply:
Done!